multimedia/audioTest/audioTest/Controller/MusicPlaylist.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  MusicPlaylist.swift
+//  audioTest
+//
+//  Created by Dhanshri Pawar on 14/04/20.
+//  Copyright © 2020 Dhanshri Pawar. All rights reserved.
+//
+
+import UIKit
+
+class MusicPlaylist: UITableViewController {
+    
+    var musicPlaylistModel = MusicPlaylistModel()
+    
+    var songList : [SongDetails] = []
+    var imageToSend : String = ""
+    var songToSend : String = ""
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        songList = musicPlaylistModel.musicArray
+    }
+
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
```

Comment:
You can use extension and add tableview delegate inside that. use commenting wherever necessary